examples/post_training_quantization/torch/save_load_mobilenet_v2/requirements.txt

```diff
@@ -0,0 +1,6 @@
+fastdownload==0.0.7
+openvino-dev==2024.0
+scikit-learn
```

Comment:
```suggestion
scikit-learn==1.4.2
```

Reply:
1.4.2 version starts from python3.9, I used the commonly used 1.2.2 version